chromium/.eslintrc.json

```diff
@@ -9,8 +9,15 @@
     "sourceType": "script"
   },
   "rules": {
-    "indent": ["error", 2],
+    "indent": ["error", 2, {"outerIIFEBody": 0}],
     "no-unused-vars": "error",
     "no-undef": "error"
+  },
+  "globals": {
+    "exports": true,
+    "util": true,
+    "storage": true,
+    "background": true,
+    "rules": true
```

Comment:
`incognito` is not present.  It is not used by any other module, either.  So why export it at all?  We should either add it to this list or remove the export and keep the function scope, or keep the export and add it as a global here.  We may also consider renaming the file to keep it consistent with the export name.

Reply:
Renamed `incognito-cache-thing.js` to `incognito.js` and added `incognito` here.